### PR TITLE
Add bBot page to bot docs

### PR DESCRIPTION
**Pull request by timkinnane**
+ Fixed minor style issue where list had no spacing between following paragraph

### CODE REVIEW

**Comment by timkinnane** [comment]
P.S. Mad thanks to @JSzaszvari - I copied a lot of the content he produced for the Hubot Docker setup.

File: bots/running-a-bbot-bot/README.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+```
+
+Choose `package.json` or `Dockerfile` as the build source.
+
+Follow the link (in clipboard) to your newly created app to view logs.
+
+## Contribute to the Docs
+
+If you've developed deployment methods other common cloud platforms, please
+submit a PR with the instructions using the link below.
```
Comment (MartinSchoeler) [inline bots/running-a-bbot-bot/README.md:181]: I don't see any links below, maybe is missing?

File: bots/running-a-bbot-bot/README.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+RESPOND_TO_DM=true
+RESPOND_TO_EDITED=true
+LISTEN_ON_ALL_PUBLIC=false
+```
+
+That's it, the app will restart and attempt connection when you change the
+settings, just view the logs to resolve any connection issues.
+
+Glitch apps will usually sleep after 5 minutes, but
+[there are ways](support.glitch.com/t/not-letting-the-app-go-down/2000/13)
```
Comment (MartinSchoeler) [inline bots/running-a-bbot-bot/README.md]: This link needs the `http://` or `https://`  prefix, as it is now it breaks the link and the tests.

**Review by MartinSchoeler — CHANGES_REQUESTED** [review]
This looks good! I've just requested some small changes.
Thanks!

File: bots/running-a-bbot-bot/README.md (hunk 1 above)
Comment (timkinnane) [reply bots/running-a-bbot-bot/README.md:181]: Every page has a "Contribute to this page on GitHub" link appended, that's all it's referring to. I was just adding a preface to call out that button and prompt for specific contributions.

File: bots/running-a-bbot-bot/README.md (hunk 2 above)
Comment (timkinnane) [reply bots/running-a-bbot-bot/README.md]: Will fix now, thanks @MartinSchoeler

**Comment by timkinnane** [comment]
@MartinSchoeler made the change re link protocol. Thanks.

**Review by MartinSchoeler — APPROVED** [review]

**Comment by timkinnane** [comment]
❤️

**Pull request merged**